api/messageservice.yaml

```diff
@@ -405,6 +405,43 @@ paths:
           description: INTERNAL SERVER ERROR - server encountered unexpected condition
       security:
         - Bearer: [ ]
+  /messages/aliasWithContent/new:
+    post:
+      tags:
+        - message-controller
+      summary: 'Saves a new system message containing the provided message type in the alias and
+      some content.
+      [Authorization: Role: user, consultant, technical]'
+      operationId: saveAliasMessageWithContent
+      parameters:
+        - name: rcGroupId
+          in: header
+          required: true
+          schema:
+            type: string
+      requestBody:
+        required: true
+        content:
+          'application/json':
+            schema:
+              $ref: '#/components/schemas/AliasMessageDTO'
```

Comment:
If you are using the same DTO like in the existing one why do you need a new Endpoint for this?

Reply:
i did not want to mix new logic with existing one. since in the new endpoint we provide also the content inside the system message.